diff --git a/node_modules/typescript/lib/typescript.js b/node_modules/typescript/lib/typescript.js
index dc0fe9a..21ba50d 100644
--- a/node_modules/typescript/lib/typescript.js
+++ b/node_modules/typescript/lib/typescript.js
@@ -154238,6 +154238,7 @@ function registerCodeFix(reg) {
     }
   }
 }
+registerCodeFix({ errorCodes: [61333], getCodeActions: () => undefined })
 var errorCodeToFixesArray;
 function getSupportedErrorCodes() {
   return errorCodeToFixesArray ?? (errorCodeToFixesArray = arrayFrom(errorCodeToFixes.keys()));
